app/BUILD.bazel

```diff
@@ -575,6 +573,9 @@ kt_android_library(
         ":dagger",
         ":listeners",
         ":resources",
+        "//app/src/main/java/org/oppia/android/app/viewmodel:observable_array_list",
+        "//app/src/main/java/org/oppia/android/app/viewmodel:observable_view_model",
+        "//app/src/main/java/org/oppia/android/app/viewmodel:view_model_provider",
```

Comment:
@fsharpasharp I've added the target into the dpes of Viewmodels 

Shall I add a single target named `libs` instead of three different targets

```bazel
kt_android_library(
    name = "libs",
    srcs = [
        "ObservableArrayList.kt",
        "ObservableViewModel.kt",
        "ViewModelBridgeFactory.kt",
        "ViewModelProvider.kt",
    ],
    visibility = ["//visibility:public"],
    deps = [
        artifact("javax.inject:javax.inject:1"),
        artifact("androidx.appcompat:appcompat"),
        artifact("androidx.databinding:databinding-adapters"),
        artifact("androidx.lifecycle:lifecycle-extensions:2.2.0"),
        artifact("androidx.lifecycle:lifecycle-livedata-core:2.2.0"),
        artifact("androidx.lifecycle:lifecycle-livedata-ktx:2.2.0"),
    ],
)

```

Reply:
Let's keep them separated if possible, this makes a difference when we create intermediate targets later. We will only need to add the dependencies we use use to each target.

Reply:
Sure thanks, Can you please review the pull request as all the checks are passing now